CMakeLists.txt

```diff
@@ -55,7 +55,13 @@ add_definitions(-fPIC)
 
 # Baseline compiler flags, any change here will affect all build types
 #set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror")
-set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -fno-var-tracking-assignments")
+
+
+IF (${DISTRO} STREQUAL "Ubuntu_17.04")
```

Comment:
Please add a comment about this line explaining why we have needed to add the if/else

Reply:
Sure, more or less the same as the description of this PR, a little more info about what is causing the warning in Ubuntu 17.04

Reply:
Moreover, first create an issue about 

> The real fix would be to rewrite the part of the code producing these warnings, but for now, disabling the -Werror flag makes the problem go away (warnings will still be found and give errors in other distros).

Then add the comment including the number of the issue in it

Reply:
Fixed in b4f1a1f, no issue though. Coming soon

Reply:
github issue (from TID repo) mentioned in FIXME in commit 88deb59